src/test/java/com/speaker/service/AccountServiceImplTest.java

```diff
@@ -0,0 +1,92 @@
+package com.speaker.service;
+
+import com.speaker.convertors.AccountConverter;
+import com.speaker.dto.AccountDTO;
+import com.speaker.entities.Account;
+import com.speaker.entities.CityName;
+import com.speaker.entities.CountryName;
+import com.speaker.repository.AccountRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+
+import static com.speaker.utils.AccountDTOGenerator.*;
+import static com.speaker.utils.AccountGenerator.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsIterableContainingInAnyOrder.containsInAnyOrder;
+import static org.hamcrest.core.Is.is;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class AccountServiceImplTest {
+
+    @Mock
+    private AccountRepository accountRepository;
+    @Mock
+    private AccountConverter accountConverter;
+    @InjectMocks
+    private AccountServiceImpl accountService;
+
+    @BeforeEach
+    public void prepare() {
+        when(accountRepository.findAllCountryAndCity())
+                .thenReturn(List.of(generateAccount(generateCountry(CountryName.UKRAINE, generateCity(CityName.KYIV)))
+                        .getCountry()));
+        this.accountService.init();
+    }
+
+    @Test
+    public void findAllAccounts() {
+        Account account = generateAccount(generateCountry(CountryName.UKRAINE, generateCity(CityName.KYIV)));
+        when(accountConverter.convertToAccountDTO(account))
+                .thenReturn(generateAccountDTO(generateCountryDTO(CountryName.UKRAINE, generateCityDTO(CityName.KYIV))));
+        when(accountRepository.findAll()).thenReturn(List.of(account));
+        assertThat(accountService.findAll(), containsInAnyOrder(accountConverter.convertToAccountDTO(account)));
```

Comment:
You must expect objects that you generate. If mock `accountConverter.convertToAccountDTO(` will return nothing, you will expect nothing, it's a wrong test.